src/components/bookmark-button/toast/make-bookmark-toast.tsx

```diff
@@ -47,6 +47,6 @@ export default function makeBookmarkToast(
 ) {
 	toast({
 		...generateToastMessage(action, tutorialName),
-		autoDismiss: 15000,
+		autoDismiss: 5000,
```

Comment:
subtask: [Adjust toast animation duration to 5 seconds](https://app.asana.com/0/0/1203075873125169/f)

Reply:
Looks like the toasts still aren't dismissing as expected 🤔 for me they persist indefinitely. 

I wonder if the package was updated.[ Looking at the docs here](https://react-hot-toast.com/docs/toast) it seems like there isn't an `autoDismiss` property anymore. Maybe we need to use `duration`?

Reply:
Also could we make the same update on the user progress toast also? Whatever it ends up being — they should be the same duration.

Reply:
So our local toast component should be using the `autoDismiss` prop to set the `duration`. Not sure why it isn't setting but I'll continue looking into it!

https://github.com/hashicorp/dev-portal/blob/main/src/components/toast/index.tsx#L30